packages/audio-block/src/AudioBlock.tsx

```diff
@@ -0,0 +1,205 @@
+        appBridge.openAssetChooser(
+            async (result) => {
+                setIsLoading(true);
+                updateAudioAsset(result[0]);
+                appBridge.closeAssetChooser();
+            },
+            {
+                multiSelection: false,
+                objectTypes: [AssetChooserObjectType.File],
+                extensions: AUDIO_EXTENSIONS,
+            }
+        );
+    };
+
+    useEffect(() => {
+        if (selectedFiles) {
+            setIsLoading(true);
+            uploadFile(selectedFiles[0]);
+        }
+        // eslint-disable-next-line react-hooks/exhaustive-deps
+    }, [selectedFiles]);
+
+    useEffect(() => {
+        if (droppedFiles) {
+            if (AUDIO_EXTENSIONS.includes(droppedFiles[0].name.split('.').pop() as FileExtension)) {
+                setIsLoading(true);
+                uploadFile(droppedFiles[0]);
+            }
+        }
+        // eslint-disable-next-line react-hooks/exhaustive-deps
+    }, [droppedFiles]);
+
+    useEffect(() => {
+        if (doneAll && uploadResults) {
+            updateAudioAsset(uploadResults[0]);
+        }
+        // eslint-disable-next-line react-hooks/exhaustive-deps
+    }, [doneAll, uploadResults]);
+
+    return (
+        <div data-test-id="audio-block" className={audioBlockClassNames}>
+            {audio ? (
+                <div className="tw-group">
+                    {isEditing && (
+                        <ItemToolbar
+                            onRemoveAsset={onRemoveAsset}
+                            onUploadClick={openFileDialog}
+                            onAssetChooseClick={openAssetChooser}
+                        />
+                    )}
+
+                    {isLoading ? (
+                        <div className="tw-flex tw-items-center tw-justify-center tw-h-14">
+                            <LoadingCircle />
+                        </div>
+                    ) : (
+                        <audio
+                            data-test-id="audio-block-audio-tag"
+                            key={audio.id}
+                            controls
+                            className={audioTagClassNames}
+                            controlsList="nodownload"
+                            preload="auto"
+                        >
+                            <source src={audio.genericUrl} type="audio/mp3" />
+                        </audio>
+                    )}
+                </div>
+            ) : (
+                isEditing && (
+                    <UploadPlaceholder
+                        onUploadClick={openFileDialog}
+                        onAssetChooseClick={openAssetChooser}
+                        loading={isLoading}
+                        setDroppedFiles={setDroppedFiles}
+                    />
+                )
+            )}
+            <div className="tw-flex tw-gap-4 tw-justify-between tw-w-full">
+                <div className="tw-self-stretch">
```

Comment:
```suggestion
                <div className="tw-flex-1">
```

Currently the RTE does not stretch to full width, so e.g. text-alignment is not working. with flex:1 it will stretch up to the attachments component.

Reply:
fixed within [commit](https://github.com/Frontify/guideline-blocks/pull/475/commits/270d443a55f7aca51132ff498bb0c1cdedbc3b59)